### PR TITLE
Fix update terms

### PR DESCRIPTION
**Pull request by curquiza**
Sorry, I obviously messed up with the rebase in the previous PR 😕

I took one by one all the previous suggestions (including the changes asked by @Kerollmops that we hadn't taken into account before the merge).

There were some "conflicts" in some sentences, I tried to do my best to merge the both, tell me if it's ok for you :)

### CODE REVIEW

File: advanced_guides/asynchronous_updates.md [hunk 1]
```diff
@@ -2,11 +2,11 @@
 
 MeiliSearch is an **asynchronous API**. It means that the API does not behave as you would typically expect when handling the request's responses.
 
-Some updates are put in a queue and will be executed in turn (asynchronously). In this case, the server response contains the identifier to track the execution of the update.
+Some operations are put in a queue and will be executed in turn (asynchronously). In this case, the server response contains the identifier to track the execution of the operation.
 
 ### Async flow
 
-- When making a write request (*create/update/delete*) against the search engine, it stores the writing udpate received in a queue and returns an `updateId`. With this id, the specific update execution is trackable.
+- When making a write request (*create/update/delete*) against the search engine, it stores the writing operation received in a queue and returns an `updateId`. With this id, the specific operation update is trackable.
```
Comment (bidoubiwa) [inline advanced_guides/asynchronous_updates.md]:
```suggestion
- When making a write request (*create/update/delete*) against the search engine, it stores the operation received in a queue and returns an `updateId`. With this id, the operation update is trackable.
```

File: advanced_guides/asynchronous_updates.md [hunk 2]
```diff
@@ -25,21 +25,21 @@ sequenceDiagram
...
 
 Updates returns the following information:
-* **status**: State of the update (enqueued, processed)
-* **updateId**: Id of the update
-* **type**: Information about the update type
-* **enqueuedAt**: Date at which the update has been added to the queue
-* **processedAt**: Date ate which the update has done processing.
+* **status**: The state of the operation (enqueued, processed, or failed).
+* **updateId**: The id of the update.
+* **type**: An information type of the operation.
```
Comment (Kerollmops) [inline advanced_guides/asynchronous_updates.md]:
```suggestion
* **type**: An information about the type of the operation.
```

Comment (bidoubiwa) [inline advanced_guides/asynchronous_updates.md]:
```suggestion
* **type**: The type of the operation
```

File: advanced_guides/asynchronous_updates.md [hunk 3]
```diff
@@ -25,21 +25,21 @@ sequenceDiagram
...
 
 Updates returns the following information:
-* **status**: State of the update (enqueued, processed)
-* **updateId**: Id of the update
-* **type**: Information about the update type
-* **enqueuedAt**: Date at which the update has been added to the queue
-* **processedAt**: Date ate which the update has done processing.
+* **status**: The state of the operation (enqueued, processed, or failed).
+* **updateId**: The id of the update.
+* **type**: An information about the type of the operation.
```
Comment (bidoubiwa) [inline advanced_guides/asynchronous_updates.md]:
```suggestion
* **type**: The type of the operation.
```

Comment (bidoubiwa) [reply advanced_guides/asynchronous_updates.md]: @Kerollmops  do you accept ?

**Pull request merged**